apps/web/components/organisms/newOpinionForm/newOpinionForm.tsx

```diff
@@ -0,0 +1,52 @@
+import { Button, Card, Heading } from 'ui';
+import { useAddOpinionForm } from 'organisms/newOpinionForm/useAddOpinionForm';
+import { TextArea } from 'ui/components/organisms/textArea/textArea';
+import { ReactComponent as StarIcon } from 'components/icons/gray-star.svg';
+
+export const NewOpinionForm = () => {
+	const { register, errors } = useAddOpinionForm();
+	console.log(StarIcon);
+	return (
+		<Card tag="div">
+			<div className="flex flex-col">
+				<Heading tag="h3" size="large" className="font-bold">
+					Dodaj opinię o zadaniu
+				</Heading>
+				<form>
+					<div className="pt-3">
+						<TextArea
+							{...register('opinion')}
+							isError={Boolean(errors.opinion)}
+							label="Twoja opinia"
+							errorMessage={errors.opinion?.message || ''}
+						/>
+					</div>
+					<div className="flex flex-col md:flex-row justify-between items-center pt-5">
+						<div className="flex justify-center items-center">
+							<div className="w-10 h-10 m-2">
```

Comment:
Why does it have `m-2`?
Could map over array of five elements or make a separate component out of this stars as it seems to appear in multiple places in the application.